jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <base-dialog title="Add identity" v-model="dialog" :error="error" :loading="loading">
+    <template #default>
+      <v-form>
+        <v-text-field v-model="form.instance_name" label="Identity instance name" dense></v-text-field>
+        <v-text-field v-model="form.tname" label="3bot name" dense></v-text-field>
+        <v-text-field v-model="form.email" label="Email" dense></v-text-field>
+        <v-text-field v-model="form.words" label="Words" dense></v-text-field>
+        <v-select v-model="form.explorer_type" label="Explorer type" :items="explorer_labels" dense></v-select>
+      </v-form>
+    </template>
+    <template #actions>
+      <v-btn text @click="close">Close</v-btn>
+      <v-btn text @click="submit">Add</v-btn>
+    </template>
+  </base-dialog>
+</template>
+
+<script>
+
+module.exports = {
+  mixins: [dialog],
+    data () {
+      return {
+        explorers: {
+          "Main Network": {url: "https://explorer.grid.tf", type: "main"},
+          "Test Network": {url: "https://explorer.testnet.grid.tf", type: "testnet"}
+        },
+        explorer_labels:["Main Network","Test Network"]
+      }
+    },
+    methods: {
+        submit () {
+            this.loading = true
+            this.error = null
+            if(!this.form.instance_name || !this.form.tname || !this.form.email || !this.form.words || !this.form.explorer_type){
+                this.alert("All fields required", "error")
+                this.loading = false
+            }
+            else{
+                this.$api.identities.add(this.form.instance_name, this.form.tname, this.form.email, this.form.words, this.explorers[this.form.explorer_type].type).then((response) => {
+                    responseMessage = JSON.parse(response.data).data
+                    if(responseMessage == "Identity with the same instance name already exists"){
+                        this.alert(responseMessage, "error")
```

Comment:
please change this to
this.error = "All fields required"

Reply:
updated in https://github.com/threefoldtech/js-sdk/pull/261/commits/66f2c240868f4fc596152d45ba4e7659d172d343